.gitignore

```diff
@@ -171,3 +171,6 @@ node_modules
 
 # Compliance/trestle related
 docs/compliance/.trestle/cache
+
+src/migrationdata/*
```

Comment:
This change is temporary. This is here to avoid accidentally committing anything to the repo while testing